docs/source/user-guide/sql/dml.md

```diff
@@ -39,7 +39,9 @@ TO '<i><b>file_name</i></b>'
 clause is not specified, it will be inferred from the file extension if possible.
 
 `PARTITIONED BY` specifies the columns to use for partitioning the output files into
```

Comment:
Should we also update the list of options ni https://datafusion.apache.org/user-guide/sql/write_options.html to include the this new one?

Reply:
I think that would be a good addition.

Reply:
Added as part of my last commit.